src/ppx/grfx/grfx_pipeline.cpp

```diff
@@ -97,6 +98,21 @@ grfx::BlendAttachmentState BlendAttachmentState::BlendModePremultAlpha()
     return state;
 }
 
+grfx::BlendAttachmentState BlendAttachmentState::BlendModeBenchmarkDisableOutput()
+{
```

Comment:
probably do not need `Benchmark` in the name, this blendmode can be used for other apps